torch/_inductor/codegen/cpp.py

```diff
@@ -1657,7 +1656,7 @@ def store_reduction(self, name, index, value):
                     ]
                 else:
                     raise AssertionError(
-                        f"Unsupported reduction type {reduction_type} from {dtype} to {out_dtype}"
```

Comment:
What's wrong with this fstring?

Reply:
`reduction_type` was not defined anywhere in this function